azure-pipelines.yml

```diff
@@ -0,0 +1,14 @@
+pool:
+  vmImage: 'ubuntu-16.04'
+
+steps:
+  - task: Gradle@2
+    inputs:
+      workingDirectory: ''
+      gradleWrapperFile: 'gradlew'
+      javaHomeOption: 'JDKVersion'
+      jdkVersionOption: '1.8'
+      jdkArchitectureOption: 'x64'
+      publishJUnitResults: false
+      testResultsFiles: '**/TEST-*.xml'
+      tasks: 'test'
```

Comment:
new line :wink:

Reply:
will this pipe publish to the azure artifacts?

Reply:
yup, this yaml is a work in progress, starting from building, and then next steps is build jar and publish